src/hotspot/share/oops/instanceKlass.cpp

```diff
@@ -980,15 +980,11 @@ void InstanceKlass::add_initialization_error(JavaThread* current, Handle excepti
   // If the initialization error is OOM, this might not work, but if GC kicks in
   // this would be still be helpful.
   JavaThread* THREAD = current;
-  Handle cause = java_lang_Throwable::get_cause_with_stack_trace(exception, THREAD);
-  if (HAS_PENDING_EXCEPTION || cause.is_null()) {
-    CLEAR_PENDING_EXCEPTION;
-    return;
-  }
```

Comment:
We still need to return if we got null.

Reply:
Thanks, I was mistaken  about my corrections in calling `JavaCalls::call_virtual(`, and thought that create_initialization_error always return a non-null handle()